packages/design-system-react/src/components/AvatarFavicon/AvatarFavicon.types.ts

```diff
@@ -21,5 +21,7 @@ export type AvatarFaviconProps = Omit<
      * Optional prop to pass to the underlying img element
      * Useful for overriding the default alt text which is the dapp name
      */
-    imageProps?: ComponentProps<'img'>;
+    imageProps?: ComponentProps<'img'> & {
+      'data-testid'?: string;
+    };
```

Comment:
See this comment https://github.com/MetaMask/metamask-design-system/pull/674/files#r2094696645